metricbeat/docs/modules/rabbitmq.asciidoc

```diff
@@ -21,12 +22,7 @@ in <<configuration-metricbeat>>. Here is an example configuration:
 ----
 metricbeat.modules:
 - module: rabbitmq
-  metricsets: ["node", "queue", "connection"]
```

Comment:
Should we just make the other 2 metricsets beta?

Reply:
Done.